### ISSUE TITLE
OpenAI API Response was: Error: Cannot find module '../utils/promptLayer'...

### ISSUE THREAD

**Issue by terese0815** [davila7/code-gpt-docs#154]
OpenAI API Response was: Error: Cannot find module '../utils/promptLayer' Require stack:

/home/stefan/.vscode-server/extensions/danielsanmedium.dscodegpt-2.1.9/src/clients/openai_client.js
/home/stefan/.vscode-server/extensions/danielsanmedium.dscodegpt-2.1.9/src/extension.js
/home/stefan/.vscode-server/bin/b3e4e68a0bc097f0ae7907b217c1119af9e03435/out/vs/loader.js
/home/stefan/.vscode-server/bin/b3e4e68a0bc097f0ae7907b217c1119af9e03435/out/bootstrap-amd.js
/home/stefan/.vscode-server/bin/b3e4e68a0bc097f0ae7907b217c1119af9e03435/out/bootstrap-fork.js

This is happening after every response when I use it via VSCode remote extension. Local all fine.

**Comment by gitforwebwork** [comment]
I accidentally added the OpenAI API key as the Prompt Layer API key, which caused the same error. It was gone, after I removed the Prompt Layer API key.
![Selection-1623](https://github.com/davila7/code-gpt-docs/assets/12074418/fb9361e4-4946-40e2-8053-80164122fa88)

**Comment by peperojel** [comment]
Same thing happening for me in Linux.

```
OpenAI API Response was: Error: Cannot find module '../utils/promptLayer'
Require stack:
- /home/ppe/.vscode/extensions/danielsanmedium.dscodegpt-2.1.20/src/clients/openai_client.js
- /home/ppe/.vscode/extensions/danielsanmedium.dscodegpt-2.1.20/src/extension.js
- /usr/share/code/resources/app/out/vs/loader.js
- /usr/share/code/resources/app/out/bootstrap-amd.js
- /usr/share/code/resources/app/out/bootstrap-fork.js
-
```

**Comment by mickomagallanes** [comment]
Same thing with me on Ubuntu 20.04. I tried restarting my pc, reinstalling the extension, and deleting the folder in .vscode but still, the extension does not load.

**Comment by davila7** [comment]
We are working on this bug. It will be fixed in version 2.1.25

**Comment by Maziar123** [comment]
2.1.26

Hello! How can I assist you today with your programming needs?OpenAI API Response was: Error: Cannot find module '../utils/promptLayer' Require stack: - /home/mz/.vscode/extensions/danielsanmedium.dscodegpt-2.1.26/src/clients/openai_client.js - /home/mz/.vscode/extensions/danielsanmedium.dscodegpt-2.1.26/src/extension.js - /opt/visual-studio-code/resources/app/out/vs/loader.js - /opt/visual-studio-code/resources/app/out/bootstrap-amd.js - /opt/visual-studio-code/resources/app/out/bootstrap-fork.js

**Comment by davila7** [comment]
@Maziar123 
The error is happening with accounts that are connected with promptlayer, we will fix this error in the next version 2.1.27

For the moment you can remove your promptlayer api key so you can use the extension.

**Comment by Maziar123** [comment]
How about removing this key?
